src/VoteDelegate.sol

```diff
@@ -102,7 +102,7 @@ contract VoteDelegate {
     }
 
     function reserveHatch() external {
-        require(block.number > hatchTrigger + HATCH_SIZE + HATCH_COOLDOWN, "VoteDelegate/cooldown-not-finished");
+        require(block.number >= hatchTrigger + HATCH_SIZE + HATCH_COOLDOWN, "VoteDelegate/cooldown-not-finished");
```

Comment:
So are we finally changing this?

Reply:
Don't see a big reason to insist here.
They are right that the triggering block could be viewed as still part of the cooldown period (as you can still lock in it and the blocking is only on the next block).
Looking at it that way it makes sense to use `>=`.

The 5/25 ratio thing is more of a nicety/bonus here.